### ISSUE TITLE
during cluster create configure-calico sometimes fail

### ISSUE THREAD

**Issue by mad01** [kubernetes/kops#2589]
When i have been creating new clusters with the following command. The calico config job fails. I have tested this with the latest master `968e0817fc` and the 1.6 release still the same problem. Also tested to bump to latest release of all the calico containers and after that the job works and is not failing. 

```bash 
kops create cluster \
  --image=coreos.com/CoreOS-stable-1235.9.0-hvm \
  --zones eu-west-1a,eu-west-1b,eu-west-1c \
  --master-zones eu-west-1a,eu-west-1b,eu-west-1c \
  --authorization RBAC \
  --networking calico \
  --kubernetes-version 1.6.3 \
  --topology private \
  --cloud aws \
  --bastion \
  --yes \
  --name ${NAME}
```

the following updated versions of calico docker images were used when it worked. 
```yaml
image: calico/node:v1.2.1
image: calico/cni:v1.8.3
image: calico/kube-policy-controller:v0.6.0
image: calico/ctl:v1.2.1
```

**Comment by chrislovecnm** [comment]
Can we get logs from k8s and the virtual machines?

cc @caseydavenport

**Comment by caseydavenport** [comment]
Yes, we should update this to use the latest Calico release which has some fixes related to Kubernetes v1.6.x.

CC @ozdanborne @heschlie

**Comment by mad01** [comment]
@chrislovecnm i have unfortunately taken down the cluster. However the issue happened (3 of 3) times i created the cluster with that command i posted with the latest release 1.6 it's likely the same if i created again.
